src/main/webapp/app/shared/consistency-check/consistency-check.component.html

```diff
@@ -24,7 +24,7 @@ <h4 class="modal-title" jhiTranslate="artemisApp.consistencyCheck.title">Consist
                 <div *ngIf="inconsistencies != undefined && inconsistencies.length > 0">
                     <span class="h1 text-danger"><fa-icon [icon]="'times'"></fa-icon></span>
                     <br />
-                    <span jhiTranslate="artemisApp.consistencyCheck.errorsFound">No errors found</span>
+                    <span jhiTranslate="artemisApp.consistencyCheck.errorsFound">Following inconsistencies have been found</span>
```

Comment:
I hope you don't hate me already :D
I have found one, hopefully, last thing:
Please recheck the translation keys. Since you renamed "errors" to "inconsistencies", it makes sense to have the keys named similarily

Reply:
Actually I was thinking it was the opposite, as you're right on this one too 😅 
Fixed this now, thanks 👍